sample-app/main.go

```diff
@@ -15,15 +15,11 @@ import (
 func main() {
 	ctx := context.Background()
 
-	traceloop := sdk.NewClient(config.Config{
-		BaseURL: "api-staging.traceloop.com",
+	traceloop := sdk.NewClient(ctx, config.Config{
```

Comment:
personally i wouldnt put config in a separate package.. id keep it in the sdk so u dont need to import that many files and poisoning the namespace